.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test_scripts"]
+	path = test_scripts
```

Comment:
nitpick: Could we name this directory something like `tod` to keep things more specific?